openrouteservice/src/main/java/org/heigit/ors/fastisochrones/AbstractIsochroneAlgorithm.java

```diff
@@ -0,0 +1,161 @@
+/*  This file is part of Openrouteservice.
+ *
+ *  Openrouteservice is free software; you can redistribute it and/or modify it under the terms of the
+ *  GNU Lesser General Public License as published by the Free Software Foundation; either version 2.1
+ *  of the License, or (at your option) any later version.
+
+ *  This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *  See the GNU Lesser General Public License for more details.
+
+ *  You should have received a copy of the GNU Lesser General Public License along with this library;
+ *  if not, see <https://www.gnu.org/licenses/>.
+ */
+package org.heigit.ors.fastisochrones;
```

Comment:
Refactoring the old isochrones algorithm is not a part of implementing the fast isochrones algorithm. Indeed a refactoring whereas the classes would be combined in a common superclass would be possible. This would need to be adressed in a separate project as it is not a small thing. The same applies to other parts of this code, too.